llvm-project/build.sh

```diff
@@ -0,0 +1,3 @@
+#!/usr/bin/env sh
+docker build --tag clangbuiltlinux/llvm-project:latest .
+docker cp llvm-project:/usr/local/bin/clang-14 clang
```

Comment:
Yeah, I don't expect to use these shell scripts as part of CI; they're more so for local testing.  Now that they're a bit simpler, I probably could just run them locally and even not check them in.

For now, I guess it is descriptive of how to get files OUT of a Docker container, since that's something I had to find out very early on in this investigation.

Perhaps a readme would be a better place for that info though. I will `git rm --cached` this file.

Reply:
Sure; I think having some script to run locally is useful so I don't think this needs to be removed. What might be more useful is a script that builds everything in one shot, rather than individual scripts for each project, but eh.

Reply:
In that case, having a global shell script invoke smaller shell scripts does give us flexibility to test out individual pieces of the pipeline.  Though I think another part of me would just like to avoid shell scripts as much as possible.

Reply:
> In that case, having a global shell script invoke smaller shell scripts does give us flexibility to test out individual pieces of the pipeline.

That is a good point.

> Though I think another part of me would just like to avoid shell scripts as much as possible.

Sure, I can understand that concern. An alternative would be a `Makefile` so that we get dependency descriptions for free?

We will need some way to easily test all of this locally though so I am not really sure we can get away from the scripting business but I am open to ideas/opinions otherwise.